src/queuing/queue_manager_interface.h

```diff
@@ -80,7 +80,7 @@ class queue_manager_interface
 	virtual request_ptr worker_finished(worker_ptr worker) = 0;
 
 	/**
-	 * Mark the current request of a worker as cancelled and do not assign it another request.
+	 * Mark the current request of a worker as cancelled and do not (re)assign it to another worker.
```

Comment:
While you're at it, why not make it clear that reassigning the job is a responsibility of the caller?

Reply:
That is two lines below :)

Reply:
Indeed it is, my bad